docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -109,30 +109,189 @@ Specific kinds of architecture guides include:
 
-## General style rules
+## General guidance for engaging content
 
-Please refer to this style guide when determining how address questions about
-English grammar, usage, and so on. Since many of these rules have equally
-justifiable alternatives, a style guide prevents debates over these questions
-from getting in the way of documenting Teleport.
+There are three keys to writing content that engages the audience:
 
-<Notice type="tip">
+- Use the *second person* point of view to directly address the reader.
+- Use an *active voice* and *present tense* whenever possible.
+- Use a *conversational tone* that is not too formal or too chummy.
 
-If a commonly debated style question does not have a resolution in this guide
-(e.g., the Oxford comma), all we ask is that you keep your style consistent
-within a particular page to maintain a professional polish.
+### Point of view
 
-</Notice>
+In most cases, address the reader directly. For tutorials, use either first person 
+plural—we, us, our, ours—or second person point of view. Because tutorials provide a 
+more guided approach to a topic, using the first person plural is a more natural and 
+commonly-accepted practice than in other types of documentation.
 
-### Voice
+Use the first person point of view sparingly and with intention. When overused, the 
+first person narrative can overwhelm the sense of a shared experience and obscure the 
+reader's journey.
 
-The documentation should be technically precise and directed toward a technical
-audience, e.g., application developers, site reliability engineers, and security
-engineering team leads. 
+Don't use “I” or “me” unless it appears in the text of a user interface element.
 
-Even when describing Teleport generally, we should emphasize specific technical
-capabilities over broad statements of benefit. The aim is not to pique the
-audience's interest, but to provide information.
+Avoid using “we” unless you have a really good rationale for it. For example, if you 
+are documenting a recommended setting or practice, using “We recommend...” is ambiguous
+("we" being who exactly?) and potentially dangerous if something goes wrong. If there
+are industry best practices or recommendations, it's okay to indicate that instead of 
+using the royal or editorial "we" in documentation.
+
+### Passive constructions
+
+In spite of the axiom to never use the passive voice, there are situations where a 
+passive sentence structure might be appropriate. Don't twist a sentence into knots just 
+to avoid a passive construction. Passive voice does have its place, but be wary of using it.
+
+When writing about software, it's often tempting to describe what's happening from the 
+code point of view. However, there's almost always a human being with a goal or task to 
+complete who is initiating the activity that the software is executing. If you keep this 
+human presence in mind, your writing will be more dynamic, easier to follow, and more 
+interesting to read.
```

Comment:
I would put this paragraph first in this section and include examples, clarifying that the default would be to avoid passive voice for prose that's more dynamic, easier to follow, etc., but not so much so that you would tie a sentence in knots. We should include examples because this isn't an idea that all docs authors are familiar with.